### DISCUSSION TITLE
How to stop text shifting on a label?

### DISCUSSION THREAD

**Discussion by Momijie** [elkowar/eww#1041] [Q&A]
I was copying out this example here, and for the cpu part, it shifts its icon to the left whenever the cpu number goes up. is there a way to have it not do this?
https://github.com/druskus20/eugh/blob/master/polybar-replacement/eww.yuck

**Comment by MyGitHubBlueberry** [comment]
You may try to put label and cpu percentage in a box as children and play with box's options.

**Comment by w-lfchen** [comment]
The `icon-module` widget contains a box with `space-evenly` set to `false`. This results in the label expanding as needed.
You can try setting a fixed width using `:width <value>` on the label [here](https://github.com/druskus20/eugh/blob/6c22be6daafe490880c9b92e7e417c485d7af076/polybar-replacement/eww.yuck#L102).
Note that if the label exceeds that width, shifting will still happen.
